geany.nsi.in

```diff
@@ -124,13 +125,17 @@ Section "!Program Files" SEC01
 
 	SetOutPath "$INSTDIR\bin"
 	File "${RESOURCEDIR}\bin\Geany.exe"
-	File "${RESOURCEDIR}\bin\Geany*.dll"
+	File "${RESOURCEDIR}\bin\*Geany*.dll"
 
 	SetOutPath "$INSTDIR\data"
 	File "${RESOURCEDIR}\data\GPL-2"
 	File "${RESOURCEDIR}\data\file*"
 	File "${RESOURCEDIR}\data\geany.glade"
+!if ${GTK_VERSION} > 2
```

Comment:
Last thing: I noticed this check doesn't work.
The installer will go into the GTK3 branch everytime, even if $GTK_VERSION is 2.x. probably because comparing the version string against an integer.

I couldn't find any easy solution after a quick try. Something like this should work in theory, IMO:
```
StrCpy $0 "${GTK_VERSION}" 1
!if ${0} == "2"
```
but it doesn't work too.
The idea with the above check was to only compare the first character of $GTK_VERSION.

There is a VersionCompare function in NSIS but it is not completely easy to use and need some helping code around. And it can't be used on the top-level, only in sections or functions, so we would need to use it in both occurrences of the GTK2/3 check.